src/pages/networks/NetworkDetailOverview.tsx

```diff
@@ -0,0 +1,184 @@
+  const { project } = useParams<{ project: string }>();
+
+  if (!project) {
+    return <>Missing project</>;
+  }
+
+  const { data: networkState, isLoading } = useQuery({
+    queryKey: [
+      queryKeys.projects,
+      project,
+      queryKeys.networks,
+      network.name,
+      queryKeys.state,
+    ],
+    queryFn: () => fetchNetworkState(network.name, project),
+  });
+
+  const updateContentHeight = () => {
+    updateMaxHeight("network-overview-tab");
+  };
+  useEffect(updateContentHeight, []);
+  useEventListener("resize", updateContentHeight);
+
+  const usageCount = network.used_by?.length ?? 0;
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  const data: Record<string, LxdUsedBy[]> = {
+    instances: filterUsedByType("instances", project, network.used_by),
+    profiles: filterUsedByType("profiles", project, network.used_by),
+  };
+
+  return (
+    <div className="network-overview-tab">
+      <Row className="section">
+        <Col size={3}>
+          <h2 className="p-heading--4">General</h2>
+        </Col>
+        <Col size={7}>
+          <table>
+            <tbody>
+              <tr>
+                <th className="p-muted-heading">Name</th>
+                <td>
+                  <ItemName item={network} />
+                </td>
+              </tr>
+              <tr>
+                <th className="p-muted-heading">Description</th>
+                <td>{network.description ? network.description : "-"}</td>
+              </tr>
+              <tr>
+                <th className="p-muted-heading">Type</th>
+                <td>{network.type}</td>
+              </tr>
+              <tr>
+                <th className="p-muted-heading">State</th>
+                <td>{network.status}</td>
+              </tr>
+              <tr>
+                <th className="p-muted-heading">IPv4</th>
+                <td>{network.config["ipv4.address"]}</td>
+              </tr>
+              <tr>
+                <th className="p-muted-heading">IPv6</th>
+                <td>{network.config["ipv6.address"]}</td>
+              </tr>
+            </tbody>
+          </table>
+        </Col>
+      </Row>
+      <Row className="section">
+        <Col size={3}>
+          <h2 className="p-heading--4">Status</h2>
+        </Col>
+        <Col size={7}>
+          <table>
+            <tbody>
+              <tr className="list-wrapper">
+                <th className="p-muted-heading">RX</th>
+                <td>
+                  {humanFileSize(networkState?.counters.bytes_received ?? 0)} (
+                  {networkState?.counters.packets_received} packets)
+                </td>
+              </tr>
+              <tr className="list-wrapper">
+                <th className="p-muted-heading">TX</th>
+                <td>
+                  {humanFileSize(networkState?.counters.bytes_sent ?? 0)} (
+                  {networkState?.counters.packets_sent} packets)
```

Comment:
These should be guarded against `null`ish values, providing a suitable placeholder:

![image](https://github.com/canonical/lxd-ui/assets/56583786/9d4c7d4b-a9f7-4e38-98e5-0d8dedaa1002)

Reply:
oh good catch.